lib/archethic/mining/validation_context.ex

```diff
@@ -1137,7 +1154,8 @@ defmodule Archethic.Mining.ValidationContext do
          transaction: tx
        }) do
     case pow do
-      "" ->
+      <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0, 0, 0>> ->
```

Comment:
Suggestion: Probably a check with the context's error might be useful, for example if the error is "Unknown origin" and the pow is empty, then the check is :ok.
And otherwise (proof of work empty and not errors as "Unknown origin", the proof of work should be redone.